.github/workflows/check-code-quality.yml

```diff
@@ -0,0 +1,21 @@
+name: check-code-quality
+on:
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Do we also want to run some checks on master everytime a new commit is added there?

So now we run the check on the feature-branch in the PR..
But maybe we also want to run it one more time after you merged to master... 

Or do we want to run a different job there? Which also deploys to pub.dev?

Reply:
Maybe in the future we want to add one more check here in this job.. somehing that does a dry-run-publish to pub.dev.. ? to check that all your changes are also valid for a publish to pub.dev? 

But if we want to add this, then I guess that is a future thing to add..
I just wanted to write down my thought

Reply:
I added the "run on push to master" option